tests/test_redirect.py

```diff
@@ -161,50 +161,50 @@ def test_redirect_firefox_home_the_product(self, mozwebqa):
     def test_notes_redirects_to_firefox_notes(self, mozwebqa):
         url = mozwebqa.base_url + "/firefox/notes/"
         response = requests.get(url)
-        Assert.contains("/firefox/", response.url)
-        Assert.contains("/releasenotes/", response.url)
+        assert '/firefox/' in response.url
+        assert '/releasenotes/' in response.url
 
     @nondestructive
     def test_all_older_redirect(self, mozwebqa):
         url = mozwebqa.base_url + "/firefox/all-older.html"
         response = requests.get(url)
-        Assert.contains("/firefox/new", response.url)
+        assert '/firefox/new' in response.url
 
     @nondestructive
     def test_old_firstrun_redirect(self, mozwebqa):
         url = mozwebqa.base_url + "/en-US/projects/firefox/3.6.13/firstrun/"
         response = requests.get(url)
-        Assert.not_equal(response.status_code, 404)
+        assert not requests.codes.not_found == response.status_code
```

Comment:
We should discuss and come up with a consistent approach for our style guide. I don't have a strong preference.